README.md

```diff
@@ -83,6 +83,20 @@ if variant.enabled {
 }
 ```
 
+### Available options
+
+The Unleash SDK takes the following options:
+
+| option            | required | default                   | description                                                                                                                                      |
+|-------------------|----------|---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| unleashUrl        | yes | n/a                       | The Unleash Proxy URL to connect to. E.g.: `https://examples.com/proxy`                                                                         |
+| clientKey         | yes | n/a                       | The Unleash Proxy Secret to be used                                                                                                             | 
+| appName           | no | unleash-swift-client       | The name of the application using this SDK. Will be used as part of the metrics sent to Unleash Proxy. Will also be part of the Unleash Context. |
+| environment       | no | default                    | The name of the environment using this SDK. Will be used as part of the metrics sent to Unleash Proxy. Will also be part of the Unleash Context. | 
```

Comment:
should ddefault be `default` or `development`? Cause by default `default` is deprecated:
![image](https://github.com/Unleash/unleash-proxy-client-swift/assets/455064/fa3ac608-163e-4b77-83f0-026efb990f11)

Reply:
it's the same as JS SDK